nilearn/decoding/space_net.py

```diff
@@ -468,9 +468,10 @@ def path_scores(solver, X, y, mask, alphas, l1_ratios, train, test,
             w_[support] = best_w
         best_w = w_
 
+    # TODO: do something with Xmean
     if len(best_w) == n_features:
-        if Xmean is None:
-            Xmean = np.zeros(n_features)
+        # if Xmean is None:
```

Comment:
I'd be in favor of totally removing Xmean from the function `path_scores` since it is unused and undocumented

Reply:
Agreed. i've left `# TODO: do something with Xmean`, hope that's OK with you.